packages/python/plotly/plotly/basedatatypes.py

```diff
@@ -3318,6 +3318,8 @@ def to_plotly_json(self):
         """
         Convert figure to a JSON representation as a Python dict
 
+        Note: May include some JSON-invalid data types, use the `PlotlyJSONEncoder` util when encoding.
```

Comment:
```suggestion
        Note: May include some JSON-invalid data types, use the `PlotlyJSONEncoder` util
        or the `to_json` method to encode to a string.
```
(and similar in the other one)

Reply:
Fixed in commit [43d0432](https://github.com/plotly/plotly.py/pull/4301/commits/43d0432805c6b4e406b7e68e688d2f0df3c6c304)